src/without-bang-important.js

```diff
@@ -0,0 +1,23 @@
+// Module with the same interface as the core aphrodite module,
+// except that styles injected do not automatically have !important
+// appended to them.
+//
+import {injectAndGetClassName} from './inject';
+
+import {
+    StyleSheet,
+    StyleSheetServer,
+    StyleSheetTestUtils,
+} from './index.js'
+
+const css = (...styleDefinitions) => {
+    // Do not append !important to style definitions
+    return injectAndGetClassName(false, styleDefinitions);
```

Comment:
To make it so you don't need the comment above, I would recommend giving that boolean a name:

```javascript
const useImportant = false;
return injectAndGetClassName(useImportant, styleDefinitions);
```